.gitignore

```diff
@@ -16,6 +16,7 @@ compile_commands.json
 /lib/spec/spec_helper.rb
 /debug*/
 /*release*/
+/*build*/
```

Comment:
If you feel like a maint patch to the cpp-project-template to make sure it has debug, release, and build flagged to ignore, that'd be cool. I'm not sure we've been consistent here